.eslintrc.json

```diff
@@ -1,20 +1,36 @@
 {
-  "parserOptions": {
-    "ecmaVersion": 2017
-  },
   "extends": [
-    "eslint:recommended"
+    "eslint:recommended",
+    "plugin:security/recommended"
   ],
   "env": {
     "browser": true,
     "commonjs": true,
     "es6": true
   },
+  "parserOptions": {
+    "ecmaVersion": 2017
+  },
+  "plugins": [
+    "jsdoc",
+    "json",
+    "security"
+  ],
   "root": true,
   "rules": {
+    "jsdoc/check-types": "error",
+    "security/detect-non-literal-fs-filename": "off",
+    "security/detect-object-injection": "off",
+
+    "eqeqeq": "error",
     "indent": ["error", 2, {"SwitchCase": 1, "VariableDeclarator": {"var": 2, "let": 2, "const": 3}}],
     "linebreak-style": ["error", "unix"],
+    "no-var": "off",
+    "no-warning-comments": "warn",
+    "prefer-const": "off",
```

Comment:
Another good rule, but may be a bit trickier to enable, unless we prevent the usage of multiple variable declarations with a single `let`/`const`. Naturally, there's an ESLint rule for that, [**one-var**](https://eslint.org/docs/rules/one-var). If we're interested, I could enable+fix the **one-var** in a future PR, then try enabling **prefer-const** and **no-var** rules.